ps_menutoplinks.class.php

```diff
@@ -67,6 +67,16 @@ public static function getLinkLang($id_linksmenutop, $id_shop)
         return array('link' => $link, 'label' => $label, 'new_window' => $new_window);
     }
 
+    public static function updateUrl($link)
+    {
+        for($i = 1; $i <= count($link); $i++) {
+            if (substr($link[$i], 0, 4) === "www.") {
```

Comment:
What if my url does not contain `www`?
I think it should be better to test if `http://` is missing, no?